internal/cmd/internal/olmv1/printing.go

```diff
@@ -0,0 +1,78 @@
+package olmv1
+
+import (
+	"cmp"
+	"fmt"
+	"os"
+	"slices"
+	"text/tabwriter"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/duration"
+
+	catalogdv1 "github.com/operator-framework/catalogd/api/v1"
+	olmv1 "github.com/operator-framework/operator-controller/api/v1"
+)
+
+func printFormattedOperators(extensions ...olmv1.ClusterExtension) {
+	tw := tabwriter.NewWriter(os.Stdout, 3, 4, 2, ' ', 0)
+	_, _ = fmt.Fprint(tw, "NAME\tINSTALLED BUNDLE\tVERSION\tSOURCE TYPE\tINSTALLED\tPROGRESSING\tAGE\n")
+
+	// sort by name
+	slices.SortFunc(extensions, func(a, b olmv1.ClusterExtension) int {
+		return cmp.Compare(a.Name, b.Name)
```

Comment:
It's been a little while, but I'm pretty sure we use blang.semver for bundle versions, and masterminds.semver for the ranges (but I'm also pretty sure that the regex is functionally different _only_ for ranges and wouldn't otherwise matter).

```suggestion
	slices.SortFunc(extensions, func(a, b olmv1.ClusterExtension) int {
		return cmp.OR(
		    cmp.Compare(a.Name, b.Name)
		    bsemver.MustParse(a.Status.Install.Bundle.Version).Compare(bsemver.MustParse(b.Status.Install.Bundle.Version))
		    )
```
... or similar

Reply:
AFAIR it might be the other way around or at least we should be using blang for ranges, but I agree that it shouldn't which lib we use to parse the version here.

I went with blang because it was already an indirect dependency because of olmv0. If someone has an opinion why blang shouldn't be used here, we can switch when we deprecate olmv0.